File: api/Controllers/Models/Entities/Delivery.cs
using System;
using System.ComponentModel.DataAnnotations;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;
using System.ComponentModel.DataAnnotations.Schema;
using Microsoft.AspNetCore.Http;

namespace IPKP___API.Controllers.Models.Entities
{
    public class Delivery
    {
        [Key]
        public Guid Delivery_ID { get; set; }        

        [StringLength(255)]
        public string Delivery_Status { get; set; }

        public DateTime DateDelivered { get; set; }

        //ignore waybill
        public string Way_Bill_File { get; set; }

        //Address Foreign key
        [ForeignKey(nameof(Delivery_Address_ID))]
        public Guid Delivery_Address_ID { get; set; }


        //Company Foreign key
        [ForeignKey(nameof(Delivery_Company_ID))]
        public Guid Delivery_Company_ID { get; set; }

    }
}

File: admin/src/app/Models/delivery.ts
import { Delivery_Company } from "./deliverycompany"
import { DeliveryAddress } from "./deliveryaddress"

export class Delivery{
    delivery_ID!: string
    delivery_Price!: number
    tracking_Number!: number

    dateDelivered!: Date

    delivery_Address_ID!: string    
    delivery_Address!: DeliveryAddress

    delivery_Company_ID!: string
    delivery_Company!: Delivery_Company
}